### PR TITLE
[quest] 25237 Tears of Stone - added

### LINKED ISSUE #6461: Quest 25237 Tears of Stone showing as avalable when it is not
### Description

https://www.wowhead.com/cata/quest=25237/tears-of-stone

Also has an alliance version.

WoWScrnShot_100424_134710

### Questie version

v10.5.1

### Game flavor

Cata

### PR DESCRIPTION
**Pull request by TorpedoCXC**
Also fixed the alliance version (25396) of the quest.

## Issue references

Fixes #6461

### CODE REVIEW

File: Database/Corrections/cataQuestFixes.lua [hunk 1]
```diff
@@ -3362,6 +3362,10 @@ function CataQuestFixes.Load()
             [questKeys.preQuestSingle] = {},
             [questKeys.objectives] = {{{39464,nil,Questie.ICON_TYPE_EVENT}}},
         },
+        [25237] = { -- Tears of Stone
+            [questKeys.requiredRaces] = raceKeys.ALL_HORDE,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:3366]:
```suggestion
```

File: Database/Corrections/cataQuestFixes.lua [hunk 2]
```diff
@@ -3558,6 +3562,7 @@ function CataQuestFixes.Load()
             [questKeys.nextQuestInChain] = 25770,
         },
         [25396] = { -- Tears of Stone
+            [questKeys.requiredRaces] = raceKeys.ALL_ALLIANCE,
```
Comment (BreakBB) [inline Database/Corrections/cataQuestFixes.lua:3565]:
```suggestion
```

**Review by BreakBB — COMMENTED** [review]
Both quests are actually already correctly marked as Alliance and Horde quest, in https://github.com/Questie/Questie/blob/master/Database/Cata/cataQuestDB.lua 🤔

**Comment by TorpedoCXC** [comment]
You are right @BreakBB. I did not check the file https://github.com/Questie/Questie/blob/master/Database/Cata/cataQuestDB.lua

I was too focused on the Corrections folder / file without thinking about the original cataQuestDB.lua file. Which of course may be correct already in that regard (faction eligibility of the quest). Sorry, I am still learning the basics of the Questie DB.

Please remove my changes on line 3366 and 3565 and only pull the change on line 3367. Thx!

**Comment by BreakBB** [comment]
Don't be sorry at all! That's what reviews are for and I know from personal experience it takes quite some time to know your way around the Questie code and/or the DB 😅

**Pull request merged**